data.tf

```diff
@@ -117,9 +118,9 @@ data "template_file" "userdata" {
 
   vars = merge({
     platform            = lookup(var.worker_groups[count.index], "platform", local.workers_group_defaults["platform"])
-    cluster_name        = aws_eks_cluster.this.name
-    endpoint            = aws_eks_cluster.this.endpoint
-    cluster_auth_base64 = aws_eks_cluster.this.certificate_authority[0].data
+    cluster_name        = aws_eks_cluster.this[count.index].name
```

Comment:
Do we need `[count.index]` here? can we just have `[0]` for simplicity and consistency?

Reply:
I updated this, to be consistent with other vars.